mindsdb/integrations/handlers/litellm_handler/README.md

```diff
@@ -1,120 +1,60 @@
+Create an AI engine from the [LiteLLM handler](https://github.com/mindsdb/mindsdb/tree/main/mindsdb/integrations/handlers/litellm_handler).
 
 ```sql
--- Create LLM engine NB you can provide api keys here or at create
--- You only need to provide key of llm you intend to use
-
-create ML_Engine litellm from litellm;
-
---simple completion with openai
-
-create model litellm_handler_simple
-predict text
-using
-engine="litellm",
-model="gpt-3.5-turbo",
-api_key="openai-api-key";
-
-select * from information_schema.models where name ="litellm_handler_simple" ;
-
-select * from litellm_handler_simple where text='Once upon a time';
-
---simple completion with anthropic
-
-create model litellm_handler_simple_anthropic
-predict text
-using
-engine="litellm",
-model="claude-2",
-api_key="anthropic-api-key";
-
-select * from information_schema.models where name ="litellm_handler_simple_anthropic" ;
-
-select * from litellm_handler_simple_anthropic where text='Once upon a time';
+CREATE ML_ENGINE litellm
+FROM litellm;
+```
 
---completion using prompt_template on create
 
-create model litellm_handler_prompt_template
-predict text
-using
-engine="litellm",
-model="gpt-3.5-turbo",
-api_key="openai-api-key",
-prompt_template="write a story based on {{text}}";
+## Usage
 
-select * from information_schema.models where name ="litellm_handler_prompt_template" ;
+Here is how to create a model that answers questions:
 
-select * from litellm_handler_prompt_template where text='Once upon a time';
+```sql
+CREATE MOEL chat_model
+PREDICT text
+USING
+    engine="litellm",
+    model="gpt-4",
+    api_key="openai-api-key";
+```
 
---completion using prompt_template on predict
+Query the model to get answer:
 
-create model litellm_handler_prompt_template_predict
-predict text
-using
-engine="litellm",
-model="gpt-3.5-turbo",
-api_key="openai-api-key";
+```sql
+SELECT *
+FROM mindsdb.chat_model
+WHERE question = "How far is the moon?"
+```
 
-select * from information_schema.models where name ="litellm_handler_prompt_template_predict" ;
+Here is the output:
 
-select * from litellm_handler_prompt_template_predict 
-where text='Once upon a time' prompt_template="write a story based on {{text}}";
+```sql
++---------------------------+-------------------------------+
+|question                   |answer                         |
++---------------------------+-------------------------------+
+|How far is the moon?   |    384,400 km                     |
++---------------------------+-------------------------------+
 
+```
 
---completion using messages with batch_completion (multiple messages)
+<Tip>
 
-create model litellm_handler_messages
-predict text
-using
-engine="litellm",
-model="gpt-3.5-turbo",
-api_key="openai-api-key";
-       
-select * from information_schema.models where name ="litellm_handler_messages" ;
+**Next Steps**
+Go to the [Use Cases](https://docs.mindsdb.com/use-cases/overview) section to see more examples.
```

Comment:
As far as I am aware, we don't have any tutorials using LiteLLM; should we still include this here?

Reply:
We are using this as a general go to section